wordpress-mobile/WordPress-Android
# UI Modernization Post: Add GB icons for Post context menu

Parent #19343 

### This PR is the first in a series which will update the Post list context menu. 

The focus of the PR is to add the gutenberg icon set and apply them to the "matching" context menu item.
- The SVG's were taken directly from the GB javascript files found in this GH [repo](https://github.com/WordPress/gutenberg/tree/trunk/packages/icons/src/library) . The svg part was converted to a Vector Asset using the Android Studio tool. 
- The following file naming pattern was applied to each icon: "gb_ic_iconname" . "iconname" matches the name in the GB repo. 
- At the time of this PR there may have been other GB icons already in the code base; however there was no way to properly identify them. A few icons used the word "new" in the filename; however those files were used in conjunction with an "old/new" feature, so they weren't changed.

**Note**: This PR does **NOT** contain the new context menu groupings or copy changes.

FYI @osullivanchris - Take a look at the context icons; I took the liberty of using GB equivalent icons outside of what you have specified (redo/undo and seen actions). All will change, but I wanted a good starting point within the app so you can review. In this PR the icons are applied to their existing text. Take note of the "contributor draft viewing as contributor" there is a "submit" that uses an older icon. Also, notice that there would also be a view and publish - should we be using "globe" for both of those icons? Anyway, this is a good PR for a quick review. Next PR will apply the updated copy, icons and grouping according to figma (which will take more time) - I wanted to get the icons in so Ajesh can use for Pages work. thanks.

Published | Scheduled | Trashed
-- | -- | --
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/cfca9cfa-7db4-4ad0-b0db-7bb72106021f"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/be020a86-4364-4f88-ae00-b1304d92555d"> | <img width="250" height="500" alt="trashed" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/458f1922-2b40-4c03-aa8d-1ded0234aad6">

Contributor Pending Review viewing as contributor | Contributor Pending Review viewing as editor/owner
-- | --
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/c3f4cff4-8157-420d-95fb-66dabbf11fde"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/df05616e-f77a-40f0-b3ea-9eb4b34a4776">

Contributor Draft viewing as contributor | Contributor Draft viewing as editor/owner
-- | --
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/e683d899-e6ce-4a0f-9658-e4c2927b9907"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/467eeca7-e56d-40cb-98bb-74f6995763fe">

Drafts | Published With Blaze capabilities
-- | --
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/d1dbcc33-1b22-4a2f-a578-20cb34454bd5"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/c738d49d-d861-43ad-9b80-48738e0a1044)">


**To test:**
- Install the JP app
- Launch and sign in
- Navigate to My Site > Posts
- Tap on the "more" icon on a post row in the Published tab
- ✅ Verify the new icon set is shown
- Repeat the last two steps for each tab 

## Regression Notes
1. Potential unintended areas of impact
The icons look off

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual/visual testing

3. What automated tests I added (or what prevented me from doing so) N/A

PR submission checklist:

UI Changes testing checklist: N/A
